docs/version-compatibility.md

```diff
@@ -4,6 +4,28 @@ This file documents compatibility between versions of this project with XR relea
 In some cases extra arguments will be required to continue using older XR releases, and those cases will be documented here.
 
 
+## XR 7.8.1
+
+Supports XR 7.7.1 and 7.8.1 (the first and most recent released versions of XRd).
```

Comment:
This line doesn't make sense, should just say that this is the main version supported by the tools without any extra args/workarounds required.

Reply:
Ah actually the heading should be `v1.1`, then this does make sense, my bad :)